app/models/concerns/orchestration/dns.rb

```diff
@@ -55,7 +55,7 @@ def rebuild_dns
   end
 
   def queue_dns
-    return log_orchestration_errors unless (dns? || dns6? || reverse_dns? || reverse_dns6?) && errors.empty?
+    return log_orchestration_errors unless (dns? || dns6? || reverse_dns? || reverse_dns6? || old&.dns? || old&.dns6? || old&.reverse_dns? || old&.reverse_dns6?) && errors.empty?
```

Comment:
My initial thought was that `dns?` should return true if there's any DNS so I wonder if we should rename `dns?` to `dns4?` or even `forward_dns4?`. In a separate change of course.

Reply:
Hmmm I think it comply with our other naming conventions like `ipaddress` vs `ipaddress6`. I kinda expect it like that, I don't mind if you want to refactor this however.